tools/gen-intrinsics-specs.py

```diff
@@ -641,7 +642,8 @@ def process_db(db, classification_db):
     ... 'B01': 'Section 1.1|Section 1.1.1',
     ... 'C01': 'classX|subclassY'
     ... }
-    >>> print(process_db(intrinsics, classification))
+    >>> table_format = 'grid'
+    >>> print(process_db(intrinsics, classification, table_format))
```

Comment:
We need to test the output also of the `github` format, because we will likely get rid of the `grid` one once we have transitioned to markdown. Please duplicate this unit test to check the output when `table_format = 'github'